packages/client/modules/userDashboard/components/OrgBilling/Billing.tsx

```diff
@@ -0,0 +1,197 @@
+import styled from '@emotion/styled'
+import {Divider} from '@mui/material'
+import React, {useEffect, useState} from 'react'
+import BillingForm from './BillingForm'
+import Panel from '../../../../components/Panel/Panel'
+import Row from '../../../../components/Row/Row'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {loadStripe} from '@stripe/stripe-js'
+import {Elements} from '@stripe/react-stripe-js'
+import CreatePaymentIntentMutation from '../../../../mutations/CreatePaymentIntentMutation'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import useMutationProps from '../../../../hooks/useMutationProps'
+import {CreatePaymentIntentMutationResponse} from '../../../../__generated__/CreatePaymentIntentMutation.graphql'
+import {CompletedHandler} from '../../../../types/relayMutations'
+
+const StyledPanel = styled(Panel)({
+  maxWidth: 976
+})
+
+const StyledRow = styled(Row)({
+  padding: '12px 16px',
+  display: 'flex',
+  alignItems: 'flex-start',
+  ':nth-of-type(2)': {
+    border: 'none'
+  }
+})
+
+const Plan = styled('div')({
+  lineHeight: '16px',
+  textTransform: 'capitalize',
+  textAlign: 'center',
+  display: 'flex',
+  padding: '0px 16px 16px 16px',
+  flexWrap: 'wrap',
+  width: '50%',
+  overflow: 'hidden'
+})
+
+const Title = styled('div')({
+  color: PALETTE.SLATE_800,
+  fontSize: 22,
+  fontWeight: 600,
+  lineHeight: '30px',
+  textTransform: 'capitalize',
+  display: 'flex',
+  width: '100%',
+  padding: '8px 0px 16px 0px'
+})
+
+const Subtitle = styled('div')({
+  color: PALETTE.SLATE_800,
+  fontSize: 16,
+  fontWeight: 600,
+  lineHeight: '30px',
+  textTransform: 'capitalize',
+  display: 'flex',
+  paddingBottom: 8
+})
+
+const Content = styled('div')({
+  width: '100%'
+})
+
+const InputLabel = styled('span')({
+  display: 'block',
+  fontSize: 12,
+  fontWeight: 600,
+  textAlign: 'left',
+  paddingBottom: 4,
+  color: PALETTE.SLATE_600,
+  textTransform: 'uppercase'
+})
+
+const InfoText = styled('span')({
+  display: 'block',
+  fontSize: 12,
+  fontWeight: 600,
+  textAlign: 'left',
+  paddingBottom: 8,
+  color: PALETTE.SLATE_600,
+  textTransform: 'none'
+})
+
+const TotalBlock = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-between',
+  paddingTop: 16
+})
+
+const ActiveUserBlock = styled('div')({
+  paddingTop: 16
+})
+
+const stripePromise = loadStripe(window.__ACTION__.stripe)
+
+const Billing = () => {
+  const [clientSecret, setClientSecret] = useState('')
+  const atmosphere = useAtmosphere()
+  const {onError} = useMutationProps()
+
+  useEffect(() => {
+    const handleCompleted: CompletedHandler<CreatePaymentIntentMutationResponse> = (res) => {
+      const {createPaymentIntent} = res
+      const {clientSecret} = createPaymentIntent
+      if (clientSecret) {
+        setClientSecret(clientSecret)
+      }
+    }
+
+    CreatePaymentIntentMutation(atmosphere, {}, {onError, onCompleted: handleCompleted})
+  }, [])
+
+  const appearance = {
```

Comment:
+1 I think this is the worst way of defining these settings: it's in the component, so it gets cluttered, but it's also untyped and you have to follow the references to see where it's used. I would be happier with pasting all this right in the component.